### PR TITLE
Changelog and Readme updates

### PR DESCRIPTION
**Pull request by davideast**
@katowulf. Change-log for 1.1.4

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -22,13 +22,13 @@ In order to use AngularFire in your project, you need to include the following f
...
 ```html
 <!-- AngularJS -->
-<script src="https://ajax.googleapis.com/ajax/libs/angularjs/1.4.9/angular.min.js"></script>
+<script src="https://ajax.googleapis.com/ajax/libs/angularjs/1.5.0/angular.min.js"></script>
 
 <!-- Firebase -->
 <script src="https://cdn.firebase.com/js/client/2.4.0/firebase.js"></script>
 
 <!-- AngularFire -->
-<script src="https://cdn.firebase.com/libs/angularfire/1.1.3/angularfire.min.js"></script>
```
Comment (jwngr) [inline README.md]: This is fine to do manually, but FYI that Catapult will do this automatically.

File: changelog.txt [hunk 2]
```diff
@@ -0,0 +1,3 @@
+feature - Upgraded Angular dependency to support 1.4.x
```
Comment (jwngr) [inline changelog.txt]: I think you mean `1.5.x`.

File: changelog.txt [hunk 3]
```diff
@@ -0,0 +1,3 @@
...
+changed - Removed PhantomJS dependency from e2e tests (now run using Chrome)
```
Comment (jwngr) [inline changelog.txt]: Both of these `changed` items should be removed. The change log tells people what they need to do to update their code. These both tell people about running the repo locally, which less than 1% of users care about.

**Comment by davideast** [comment]
@jwngr Updated changelog from your comments.

**Comment by jwngr** [comment]
:ship: Ship it!

**Pull request merged**